utilities/dockerfiles/wildfly/Dockerfile

```
@@ -0,0 +1,46 @@
+FROM jboss/base-jdk:8
+
+# Set the WILDFLY_VERSION env variable
+ENV WILDFLY_VERSION wildfly-latest
+#ENV WILDFLY_SHA1 eaef7a87062837c215e54511c4ada8951f0bd8d5
+ENV JBOSS_HOME /opt/jboss/wildfly
+
+USER root
+RUN rm /etc/localtime \
+   && ln -s /usr/share/zoneinfo/Asia/Kolkata /etc/localtime
+RUN cd $HOME
+RUN yum install -y epel-release
```

Hadolint (2.12.0)

[warning] `yum clean all` missing after yum command.

(DL3032)